setup.cfg

```diff
@@ -20,3 +20,6 @@ addopts =
     --durations=10 -ra --cov-report= --tb=short
     --doctest-ignore-import-errors --junit-xml=junit-results.xml
     --color=yes --capture=sys
+filterwarnings =
+    error
+    ignore:datetime\.datetime\.utcfromtimestamp.* is deprecated and scheduled for removal in a future version.*:DeprecationWarning
```

Comment:
this is not working and I don't get it ... can't seem to find the problem locally either.

Passing `-W ignore::DeprecationWarning` on the command line works 🤷 

any idea @larsoner?

Reply:
I don't think the dot includes newlines so if the warning has them they have to be added

Reply:
```suggestion
filterwarnings =
    error
    ignore:datetime\.datetime\.utcfromtimestamp.* is deprecated and scheduled for removal in a future version.*\n.*:DeprecationWarning
```